### PR TITLE
feat(outputs): add hostname, tags, custom, and templated fields to TimescaleDB

### PR DESCRIPTION
**Pull request by alika**
**What type of PR is this?**
/kind feature

**Any specific area of the project related to this PR?**
/area outputs

**What this PR does / why we need it**:
This PR adds support for hostname, tags, custom fields, and templated fields for the TimescaleDB output.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Review by Issif — COMMENTED** [review]
I see the output fields are not added, because of the the schema. Do you think it worth to add `k8s.pod.name` and `k8s.ns.name` at least? IN prometheus for example, to avoid huge cardinality, we add these fields as labels and the custom fields.

File: README.md [hunk 1]
```diff
@@ -1236,14 +1236,36 @@ permissions to access the resources you selected to use, like `SQS`, `Lambda`,
 To use TimescaleDB you should create the Hypertable first, following this example
 
 ```sql
-CREATE TABLE falco_events (
+CREATE TABLE falcosidekick_events (
```
[the remark this answers is not in the document]
Comment (alika) [reply README.md:1239]: I'm actually changing it to be the same as the default value for `TimescaleDB.HypertableName` config -> https://github.com/falcosecurity/falcosidekick/blob/master/config.go#L407

**Comment by alika** [comment]
> I see the output fields are not added, because of the the schema. Do you think it worth to add `k8s.pod.name` and `k8s.ns.name` at least? IN prometheus for example, to avoid huge cardinality, we add these fields as labels and the custom fields.

Sure, sounds reasonable. I'll add support for output fields.

**Comment by Issif** [comment]
> > I see the output fields are not added, because of the the schema. Do you think it worth to add `k8s.pod.name` and `k8s.ns.name` at least? IN prometheus for example, to avoid huge cardinality, we add these fields as labels and the custom fields.
> 
> Sure, sounds reasonable. I'll add support for output fields.

All output fields it will be hard, because they are dynamic, but at the custom fields added statically by Falcosidekick BUT it means you need to deal with the addition of new fields after the creation of the index aka, ie, a schema update.

Tell me when it's ready, I'll review this PR at this moment. Thanks.

**Comment by alika** [comment]
> > > I see the output fields are not added, because of the the schema. Do you think it worth to add `k8s.pod.name` and `k8s.ns.name` at least? IN prometheus for example, to avoid huge cardinality, we add these fields as labels and the custom fields.
> > 
> > 
> > Sure, sounds reasonable. I'll add support for output fields.
> 
> All output fields it will be hard, because they are dynamic, but at least the custom fields added statically by Falcosidekick BUT it means you need to deal with the addition of new fields after the creation of the index, ie, a schema update.
> 
> Tell me when it's ready, I'll review this PR at this moment. Thanks.

After I looked at how [`newFalcoPayload`](https://github.com/falcosecurity/falcosidekick/blob/master/handlers.go#L78) works I understand how `OutputFields` is already populated w/`Customfields` and `Templatedfields`. I have an implementation right now that **only** adds to TimescaleDB payload if it is declared in either `Customefields` or `Templatedfields`. Just doing a bit more actual testing and I'll have this PR updated soon (<24h).

**Comment by alika** [comment]
@Issif ready for review now. I had to touch config because I was getting a panic on templated fields:
```
panic: assignment to entry in nil map

goroutine 1 [running]:
main.getConfig()
        /Users/alika/projects/ref/falcosidekick/config.go:474 +0x3eeb
main.init.0()
        /Users/alika/projects/ref/falcosidekick/main.go:89 +0xff
```

**Comment by Issif** [comment]
Sorry for the delay, I was successively sick, off and at kubecon in last weeks. Have you fixed the issue with the nil map? Can I review the whole PR now? Thanks

**Comment by alika** [comment]
> Sorry for the delay, I was successively sick, off and at kubecon in last weeks. Have you fixed the issue with the nil map? Can I review the whole PR now? Thanks

No worries. Hope kubecon was fun. I did fix this issue w/the nil map and I'm ready for a review.

**Review by Issif — APPROVED** [review]

**Pull request merged**